py/label_microservice/universal_kind_label_model.py

```diff
@@ -75,19 +72,37 @@ def predict_issue_labels(self,  title:str, body:str):
      'feature': 0.6401631832122803,
      'question': 0.2761166989803314}
     """
+    if not context:
+      context = {}
     #transform raw text into array of ints
     vec_body = self.body_pp.transform([body])
     vec_title = self.title_pp.transform([title])
 
     # make predictions with the model
+    # TODO(jlewi): Is this right? Do we want to get the default graph or
```

Comment:
I think getting the default graph works because it makes it easy to refer to that graph lazily and we are only using this one graph at the moment